module-1/lab-api-scavenger-game/your-code/challenge-3.ipynb

```diff
@@ -9,7 +9,7 @@
   },
```

Comment:
Amazing approach Alberto!! Great job. Congrats for finding the hidden message!! 
It's normal that your requests last more than 2 minutes. When you write down a script with several for loops it's quite normal that requests take some time. You must try simplify your code and run as less for loops as you can. 
I encourage you to optimize your code if you have time.